x/upgrade/test/integration_test.go

```diff
@@ -0,0 +1,201 @@
+package test
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/testutil/testnode"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	v1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/config"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+)
+
+func TestUpgrade(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping SDK integration test in short mode.")
```

Comment:
[question] do we want to differentiate this log line from https://github.com/rootulp/celestia-app/blob/516ac26aae4db6bbb71e2559297622bf04571c4c/app/test/std_sdk_test.go#L29
```suggestion
		t.Skip("skipping x/upgrade SDK integration test in short mode.")
```